### PR TITLE
Creates a pattern of style guides

### PR DESCRIPTION
**Pull request by LadyCarni**
- adds css custom properties for ease of maintenance and consistency
- creates naming conventions that are generic and easy to understand
- updates for styling (font sizes, font families, etc.) closer to Microsoft's UWP standards for readability, accessibility, and brand consistency
- organizes css into logical pieces for readability

screen shot 2018-07-23 at 11 17 12 am

screen shot 2018-07-23 at 11 18 14 am

screen shot 2018-07-23 at 11 18 27 am

### CODE REVIEW

File: docs/css/style.css [hunk 1]
```diff
@@ -1,11 +1,23 @@
...
-
-/* Animations */
-
 :root {
-  --blue: #05a6f0;
-  --yellow: #ffba08;
-  --green: #6cc644;
+  /* color defaults */
+  --primary-color: #05a6f0;
+  --secondary-color: #FFBA08;
```
Comment (prichodko) [inline docs/css/style.css]: Although I understand what you were trying to do, wouldn't `--brand-blue` and `--brand-yellow` make more sense?

Comment (LadyCarni) [reply docs/css/style.css]: Yeah, I did consider that. Considering the prominent use in the markup of both the yellow and blue as what looked like primary and secondary, that was the naming I chose.

I can see what you're saying though, especially for consistency and ease of understanding.

Thanks for the hasty response!

**Comment by LadyCarni** [comment]
@prichodko I tossed in some quick updates for consistency :)

**Comment by dvdsgl** [comment]
I like almost all of this except I'm not interested in using Segoe, especially not on non-Windows OSes. Can you please restore the previous typeface?

**Pull request merged**